sources/packages/forms/src/form-definitions/noticeofassessment.json

```diff
@@ -3078,43 +3228,24 @@
                     ],
                     "content": "{{typeof data.assessment?.tuitionCost === 'number' ? '$' + (data.assessment.tuitionCost).toFixed(2) : '-'}}",
```

Comment:
this is not a part of this PR @guru-aot , I think `.toFixed()` was having some issue, it was not rounding correctly, in some scenarios. remember @dheepak-aot @andrewsignori-aot ?

Reply:
Here toFixed() has been used to display as 2 decimal positions(e.g 20 as 20.00) these values are already rounded with Math.Round as per our discussion in camunda.

Reply:
I checked also on Camunda @ann-aot, the rounding there is happening as expected.
So, what @dheepak-aot is saying is when we have a value like 123 the to fixed will make it 123.00 or 123.9 will be 123.90.